crates/net/eth-wire/src/p2pstream.rs

```diff
@@ -493,13 +493,15 @@ where
 /// capabilities and the input list of locally supported capabilities.
 ///
 /// Currently only `eth` versions 66 and 67 are supported.
+// ANDREW: We should also mention that the `p2p` protocol (version 5?) is supported,
```

Comment:
```suggestion
// todo(akirillo): We should also mention that the `p2p` protocol (version 5?) is supported,
```

Nit use this format in your todo comments